runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/DokkaMultiModuleTask.kt

```diff
@@ -52,6 +52,10 @@ abstract class DokkaMultiModuleTask : AbstractDokkaParentTask() {
     val fileLayout: Property<DokkaMultiModuleFileLayout> = project.objects.safeProperty<DokkaMultiModuleFileLayout>()
         .safeConvention(DokkaMultiModuleFileLayout.CompactInParent)
 
+    @Internal
+    val moduleOrder: Property<DokkaMultiModuleOrder> = project.objects.safeProperty<DokkaMultiModuleOrder>()
+        .safeConvention(DokkaMultiModuleOrder.Default)
```

Comment:
This has a problem that such property is not included in the cache key, therefore changing just the order won't rebuild the documentation.